bin/zssServer.sh

```diff
@@ -25,9 +33,12 @@ then
 elif [ -d "$INSTANCE_DIR" ]
 then
   CONFIG_FILE="${INSTANCE_DIR}/workspace/app-server/serverConfig/server.json"
-else  
+elif [ -d "$ROOT_DIR" ]
+then
+    CONFIG_FILE="${ROOT_DIR}/components/app-server/share/zlux-app-server/defaults/serverConfig/server.json"
```

Comment:
Please add a comment here to explain:
this behavior is different from the app-server, which when INSTANCE_DIR or WORKSPACE_DIR are not defined, they create an instance area in ~/.zowe using nodejs initialization code.
In order for zss to be run in a development environment without needing nodejs, under the circumstance no env vars for the instance location are defined, zss will use the default server.json... however this is not preferred, just a backup.